servers/physics_3d/joints/cone_twist_joint_3d_sw.h

```diff
@@ -67,39 +67,39 @@ class ConeTwistJoint3DSW : public Joint3DSW {
 			Body3DSW *B;
 		};
 
-		Body3DSW *_arr[2];
+		Body3DSW *_arr[2] = { nullptr, nullptr };
 	};
 
-	JacobianEntry3DSW m_jac[3]; //3 orthogonal linear constraints
+	JacobianEntry3DSW m_jac[3] = {}; //3 orthogonal linear constraints
 
-	real_t m_appliedImpulse;
+	real_t m_appliedImpulse = 0.0;
 	Transform3D m_rbAFrame;
 	Transform3D m_rbBFrame;
 
-	real_t m_limitSoftness;
-	real_t m_biasFactor;
-	real_t m_relaxationFactor;
+	real_t m_limitSoftness = 0.0;
+	real_t m_biasFactor = 0.0;
+	real_t m_relaxationFactor = 0.0;
```

Comment:
Missed copying the real values:
```suggestion
	real_t m_biasFactor = 0.3;
	real_t m_relaxationFactor = 1.0;
```